src/view/HenvendelseList.tsx

```diff
@@ -9,19 +9,30 @@ interface Props {
 }
 
 export function HenvendelseList(props: Props) {
+    const endOfMessages = useRef<HTMLDivElement>(null);
+    const [isFirstRender, setIsFirstRender] = useState(true);
+
+    const scrolllToBottom = () => {
+        if (endOfMessages.current !== null) {
+            endOfMessages.current.scrollIntoView({ behavior: isFirstRender ? 'auto' : 'smooth' });
+            setIsFirstRender(false);
+        }
+    };
+    useEffect(scrolllToBottom, [props.henvendelseDataList.map(data => data.id)]);
```

Comment:
dependencyListen bør inneholde avhengighetene til `scrolllToBottom`, e.g `endOfMessages`,  `isFirstRender` og `setIsFirstRender`, men ikke henvendelses-data.

Reply:
Men skal den ikke `scrollToBottom` hver gang det kommer en ny melding også?
`isFirstRender` er for å unngå, når man refresher, at appen starter i toppen av henvendelsene og scroller seg nedover, men viser nederste henvendelse med en gang.

Reply:
Tar tilbake det om `props.henvendelseDataList`. ;) 
Men `endOfMessages`, `isFirstRender` og `setIsFirstRender` burde også være i dependencylisten.